engine/api/get-started.md

```diff
@@ -606,6 +608,101 @@ $ curl --unix-socket /var/run/docker.sock \
 </div>
 </div> <!-- end tab-content -->
 
+
+### Pull images with authentication
+
+Pull images, like `docker pull`, with authentication:
+
+> **Note**: Credentials are sent in the clear. Docker's official registries use
+> HTTPS. Private registries should also be configured to use HTTPS.
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#tab-pullimages-auth-python" data-group="python">Python</a></li>
+  <li><a data-toggle="tab" data-target="#tab-pullimages-auth-go" data-group="go">Go</a></li>
+  <li><a data-toggle="tab" data-target="#tab-pullimages-auth-curl" data-group="curl">Curl</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="tab-pullimages-auth-python" class="tab-pane fade in active" markdown="1">
+
+```python
+import docker
+client = docker.from_env()
+image = client.images.pull("alpine", auth_config=['username','password'])
+print image.id
+```
+
+</div>
+
+<div id="tab-pullimages-auth-go" class="tab-pane fade" markdown="1">
+
+```go
+package main
+
+import (
+	"io"
+	"os"
+  "encoding/json"
+  "encoding/base64"
```

Comment:
Mixed tabs/spaces here as well (see some other parts below that also mix tabs/spaces)